subsys/bluetooth/host/l2cap.c

```diff
@@ -1311,6 +1317,39 @@ static void le_ecred_reconf_req(struct bt_l2cap *l2cap, uint8_t ident,
 
 	l2cap_send(conn, BT_L2CAP_CID_LE_SIG, buf);
 }
+
+#if defined(CONFIG_BT_TESTING)
+static void le_ecred_reconf_rsp(struct bt_l2cap *l2cap, uint8_t ident,
+				struct net_buf *buf)
+{
+	struct bt_conn *conn = l2cap->chan.chan.conn;
+	struct bt_l2cap_ecred_reconf_rsp *rsp;
+	struct bt_l2cap_le_chan *ch;
+	uint16_t result;
+
+	if (buf->len < sizeof(*rsp)) {
+		BT_ERR("Too small ecred reconf rsp packet size");
```

Comment:
Might be beneficial to also log the value that was too small, and possibly also the expected minimum value

Reply:
I can add this but initially wanted to keep this same as for other l2cap functions

Reply:
This is a minor one, and just a suggestion. If the other L2CAP functions do the same, then my suggestion also applies to them :D